domains/scienceweek.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "I will use this domain for an event website",
+    "repo": "https://github.com/regisx001/scienceweek",
+    "owner": {
+        "username": "regisx001",
+        "email": "[email protected]"
```

```suggestion
        "email": "[email protected]"
```
Not a valid value for an email record. If you don't want to provide your email, just remove this line (and the comma on the previous line.)